source/common/network/proxy_protocol.cc

```diff
@@ -80,11 +106,7 @@ void ProxyProtocol::ActiveConnection::onReadWorker() {
   removeFromList(parent_.connections_);
 
   // TODO(mattklein123): Parse the remote port instead of passing zero.
```

Comment:
Should this PR also add the parsed remote port? The reason I left it out is that I'm not sure if anyone is expecting the value to be 0.